README.md

```diff
@@ -171,6 +171,20 @@ And finally (do not forget):
 
 		LC_ALL=C repoowner=theforeman branch=master proxy_repo=1.14 vagrant destroy $distro
 
+Extensions
+----------
+
+Discovery Image supports runtime extensions published via TFTP or HTTP.
+Those are distributed as ZIP file with shell scripts. It is also possible
+to build an image with extension built-in which is helpful for PXE-less
+environments.
+
+To do that, (follow the documentation)[https://theforeman.org/plugins/foreman_discovery/8.0/index.html#5.Extendingtheimage]
```

Comment:
To do that, create a directory structure inside of `root/opt/extension` or follow the [complete documentation](https://theforeman.org/plugins/foreman_discovery/8.0/index.html#5.Extendingtheimage). 

Text and link are off text needs `[]` - url needs `()`.